### PR TITLE
added connection string

### PR DESCRIPTION
**Pull request by ARodrigue7**

### CODE REVIEW

File: NJInsurancePlatform/appsettings.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-      "Default": "Information",
-      "Microsoft.AspNetCore": "Warning"
+    "Logging": {
+        "LogLevel": {
+            "Default": "Information",
+            "Microsoft.AspNetCore": "Warning"
+        }
+    },
+    "AllowedHosts": "*",
+    "InsureConnect": {
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/appsettings.json:9]: "ConnectionString"{} not insureconnect

File: NJInsurancePlatform/appsettings.json [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+        "value": "Server=.;Database=InsuranceCorp;TrustedConnection=True;MultipleActiveResultSets=True;" }
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/appsettings.json:10]: within connection string"InsureConnect": "

**Review by Marcus-Johnson — CHANGES_REQUESTED** [review]

File: NJInsurancePlatform/appsettings.json (hunk 1 above)
Comment (ARodrigue7) [reply NJInsurancePlatform/appsettings.json]: updated values

File: NJInsurancePlatform/appsettings.json (hunk 2 above)
Comment (ARodrigue7) [reply NJInsurancePlatform/appsettings.json]: updated values

**Pull request merged**